tests/pytests/unit/states/test_nxos.py

```diff
@@ -0,0 +1,549 @@
+"""
+    :codeauthor: Mike Wiebe <@mikewiebe>
+"""
+
+# Copyright (c) 2019 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import salt.states.nxos as nxos_state
+from tests.support.mock import MagicMock, patch
+
+
+@pytest.fixture
+def configure_loader_modules():
+    return {nxos_state: {}}
+
+
+def test_user_present_create():
+
+    """ UT: nxos module:user_present method - create """
+
```

Comment:
```suggestion
def test_user_present_create():
    """
    user_present method - create
    """

```

Reply:
And the same approach for the other docstrings please.

Reply:
Fixed